integration-tests/deployment/ccip/deploy_home_chain.go

```diff
@@ -52,6 +52,21 @@ const (
 	MaxDurationShouldTransmitAcceptedReport = 10 * time.Second
 )
 
+var (
+	// TODO: sort out why the mismatch here
+	//CCIPCapabilityId [32]byte = utils.MustHash(hexutil.Encode(MustABIEncode(`[{"type": "string"}, {"type": "string"}]`, CapabilityLabelledName, CapabilityVersion)))
```

Comment:
You shouldn't hash the hex encoded string, just the ABI-encoded bytes

Reply:
See https://github.com/smartcontractkit/chainlink/blob/a2514d725f66e2c4a66d9bd0dd211a617fa32367/core/capabilities/ccip/common/common.go#L12-L23